VagrantProvision.sh

```diff
@@ -193,89 +254,90 @@ PATH=$HOOT_HOME/bin:$PATH
 fi
 
-# Fix env var path for GDAL_DATA
 if grep -i --quiet 'gdal/1.10' /etc/default/tomcat6; then
-    echo "Fixing Tomcat GDAL_DATA env var path"
+    echo "Fixing Tomcat GDAL_DATA env var path..."
     sudo sed -i.bak s@^GDAL_DATA=.*@GDAL_DATA=\/usr\/local\/share\/gdal@ /etc/default/tomcat6
 fi
 
 # Remove gdal libs installed by libgdal-dev that interfere with
 # renderdb-export-server using gdal libs compiled from source (fgdb support)
-if [ -f /usr/lib/libgdal.* ]; then
-    echo "Removing GDAL libs installed by libgdal-dev"
+if [ -f "/usr/lib/libgdal.*" ]; then
+    echo "Removing GDAL libs installed by libgdal-dev..."
     sudo rm /usr/lib/libgdal.*
 fi
 
-# Create Tomcat context path for tile images
+mkdir -p $HOOT_HOME/ingest/processed
+chown -R vagrant:tomcat6 $HOOT_HOME/ingest
 if ! grep -i --quiet 'ingest/processed' /etc/tomcat6/server.xml; then
-    echo "Adding Tomcat context path for tile images"
+    echo "Adding Tomcat context path for tile images..."
     sudo sed -i.bak "s@<\/Host>@  <Context docBase=\"\/home\/vagrant\/hoot\/ingest\/processed\" path=\"\/static\" \/>\n      &@" /etc/tomcat6/server.xml
 fi
 
-# Allow linking in Tomcat context
 if ! grep -i --quiet 'allowLinking="true"' /etc/tomcat6/context.xml; then
-    echo "Set allowLinking to true in Tomcat context"
+    echo "Set allowLinking to true in Tomcat context..."
     sudo sed -i.bak "s@^<Context>@<Context allowLinking=\"true\">@" /etc/tomcat6/context.xml
 fi
 
-# Create directory for webapp
-if [ ! -d /usr/share/tomcat6/.deegree ]; then
-    echo "Creating directory for webapp"
-    sudo mkdir /usr/share/tomcat6/.deegree
-    sudo chown tomcat6:tomcat6 /usr/share/tomcat6/.deegree
-fi
-
-# Tweak dev environment to make tests run faster
-if [ ! -f $HOOT_HOME/hoot-services/src/main/resources/conf/local.conf ]; then
-    echo 'testJobStatusPollerTimeout=250' > $HOOT_HOME/hoot-services/src/main/resources/conf/local.conf
+if [ ! -d $TOMCAT6_HOME/.deegree ]; then
+    echo "Creating deegree directory for webapp..."
+    sudo mkdir $TOMCAT6_HOME/.deegree
+    sudo chown vagrant:tomcat6 $TOMCAT6_HOME/.deegree
 fi
 
-# Update the init.d script for node-mapnik-server
-sudo cp node-mapnik-server/init.d/node-mapnik-server /etc/init.d
-sudo chmod a+x /etc/init.d/node-mapnik-server
-# Make sure all npm modules are installed
-cd node-mapnik-server
-npm install
-cd ..
-
 # Update marker file date now that dependency and config stuff has run
 # The make command will exit and provide a warning to run 'vagrant provision'
 # if the marker file is older than this file (VagrantProvision.sh)
 touch Vagrant.marker
 
-# Build Hoot
-echo "Configuring Hoot"
-aclocal && autoconf && autoheader && automake && ./configure -q --with-rnd --with-services
+echo "Configuring Hoot..."
+echo HOOT_HOME: $HOOT_HOME
+cp conf/DatabaseConfig.sh.orig conf/DatabaseConfig.sh
+
+if [ ! -f /etc/init.d/node-mapnik-server ]; then
```

Comment:
if we don't do this always, the init.d script will not get updated if the source changes.